zeeguu/core/util/time.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime, timezone, timedelta
+
+
+SERVER_HOUR_DIFFERENCE = 1  # Frankfurt is +1 from UTC
+
+
+def get_server_time_utc():
+    # Returns the time with the time delta of the server, converted
```

Comment:
i officially hate timezones.
can you detail a bit more in the comment why do we need this?

Reply:
I have added a long doc string explaining what we discussed. It's a bit hard to put it in a small amount of words. I recently read an article in datetime and why it's a bit of a pain, and I can relate writing this.

The main point is the tz awareness of datetime objects. 

`get_server_time_utc` is unused at the moment, it's in there in case we ever need to convert the server time to UTC and have it be tz aware for a certain operation with other tz objects.